pkg/config/images.go

```diff
@@ -39,12 +21,10 @@ func openShiftVersion(imageVersion string) (string, error) {
 func (g *simpleGenerator) selectNodeImage(cs *api.OpenShiftManagedCluster) {
 	c := &cs.Config
 	c.ImagePublisher = "redhat"
-	c.ImageOffer = g.pluginConfig.TestConfig.ImageOffer
 	if c.ImageOffer == "" {
 		c.ImageOffer = "osa"
 	}
 
-	c.ImageVersion = g.pluginConfig.TestConfig.ImageVersion
 	switch g.pluginConfig.TestConfig.DeployOS {
 	case "", "rhel7":
 		c.ImageSKU = "osa_" + strings.Replace(cs.Properties.OpenShiftVersion[1:], ".", "", -1)
```

Comment:
You should set this too only if it's empty, right?

Reply:
Ditto for centos7

Reply:
This would change current flow of how we define our images. But yes, this does not look right now. Let me think about this :man_shrugging:

Reply:
I think we might just remove this all selectNodeImages and make it default for PluginConfig